tesseract_environment/src/environment.cpp

```diff
@@ -1355,6 +1352,12 @@ bool Environment::applyCommandsHelper(const Commands& commands)
         success &= applySetActiveDiscreteContactManagerCommand(cmd);
         break;
       }
+      case tesseract_environment::CommandType::ADD_TRAJECTORY_LINK:
+      {
+        auto cmd = std::static_pointer_cast<const AddTrajectoryLinkCommand>(command);
+        success &= applyAddTrajectoryLinkCommand(cmd);
+        break;
+      }

```

Comment:
Just looking for similar places it looks like this needs to be added in tesseract_environment_unit.cpp as well as some places in tesseract_qt and tesseract_ros

Reply:
Yea, I have branches on all of the other repos and some may already be PR's. After merged I will cleanup and make sure CI passes on those repos.